api/src/test/java/org/openmrs/module/openconceptlab/importer/SaverTest.java

```diff
@@ -172,6 +198,68 @@ private Name newFourthName() {
 		return fourthName;
 	}
 
+	/**
+	 * @see Saver#saveConcept(CacheService, Import, OclConcept)
+	 * @verifies add new names to concept
+	 */
+	@Test
+	public void importConcept_shouldAddNewPreferredNamesInDifferentLocales() throws Exception {

```

Comment:
My only nit with this is it would be better to name it `saveConcept()`, since that's the method being tested. I've been doing that with newer tests, but haven't (yet) migrated existing tests.